client_app/src/components/volunteer/Shifts.js

```diff
@@ -49,13 +64,28 @@ export function UpcomingShifts() {
   );
 }
 
-export function PastShifts() {
+export function PastShifts({ onImpactDataUpdate }) {
   const time_now = new Date().getTime();
   const shelters_endpoint = SERVER + "/shifts?filter_end_before=" + time_now;
+  const header = getAuthHeader();
+
+  useEffect(() => {
+    fetch(shelters_endpoint, {
+      method: "GET",
+      headers: header,
+    })
+      .then((response) => response.json())
+      .then((shifts) => {
+        const totalHours = calculateTotalHours(shifts);
+        const sheltersServed = calculateUniqueShelters(shifts);
+        onImpactDataUpdate({ totalHours, sheltersServed });
+      })
+      .catch((error) => console.log(error));
+  }, [shelters_endpoint, onImpactDataUpdate]);
+
   return (
     <div>
       <h1 className="text-center">Previous Shifts</h1>
-      {Shifts(shelters_endpoint)}
```

Comment:
I think this changed causes "Past Shifts" not to show up in the volunteer dashboard.